script/stresstest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# stresstest.sh is a helper script to run specific Go tests with x/tools/cmd/stress.
+#
+# Usage: ./script/stresstest.sh [package_path] ^TestName$ [stress options...]
```

Comment:
As this is the argument passed to `-run`, I believe it's better to document like it.

Reply:
Ack.